kratos/geometries/triangle_3d_3.h

```diff
@@ -875,24 +875,19 @@ template<class TPointType> class Triangle3D3
         const double Tolerance = std::numeric_limits<double>::epsilon()
         ) const override
     {
-        // The center of the geometry
-        const auto& r_center = this->Center();
-
         // We compute the normal to check the normal distances between the point and the triangles, so we can discard that is on the triangle
-        const CoordinatesArrayType aux_point = ZeroVector(3);
-        const array_1d<double, 3> normal = this->UnitNormal(aux_point);
+        const array_1d<double, 3> normal = this->UnitNormal(this->Center());
 
         // We compute the distance, if it is not in the pane we
         const Point point_to_project(rPoint);
         Point point_projected;
-        const array_1d<double,3> vector_points = rPoint - r_center.Coordinates();

```

Comment:
This is the direction you used and its not the normal 

I corrected it, now it does use the normal

Reply:
OK I see, I used the wrong utility. Wait a second.

Besides the algorithm is quite common, here an example:

https://stackoverflow.com/questions/9605556/how-to-project-a-point-onto-a-plane-in-3d

Reply:
I will use instead FastProject, you are computing twice the normal (performance)

Reply:
makes sense, thx
will test it before merging